### PR TITLE
Showed an example of using container parameters in route requirements

### PR DESCRIPTION
**Pull request by javiereguiluz**
Following this discussion https://github.com/symfony/symfony/pull/18599 I thought it may be useful to add @lyrixx's comment (https://github.com/symfony/symfony/pull/18599#issuecomment-212794145) as an example in the docs.

### CODE REVIEW

**Comment by lyrixx** [comment]
@javiereguiluz I like that. But I'm not 100% sure this feature existed in symfony 2.3 ; Can someone confirm it worked in 2.3 ?

**Comment by ogizanagi** [comment]
It should work in 2.3: https://github.com/symfony/symfony/blob/2.3/src/Symfony/Bundle/FrameworkBundle/Routing/Router.php#L92-L94

File: book/routing.rst [hunk 1]
```diff
@@ -808,6 +808,101 @@ Path     Parameters
 ``/es``  *won't match this route*
 =======  ========================
 
+.. tip::
+
+    The route requirements can also be defined as container parameters, which
+    comes in handy when the regular expression is very complex and used time and
+    again in your application:
```
Comment (xabbuh) [inline book/routing.rst]: I would end this paragraph with a dot.

**Comment by xabbuh** [comment]
:+1:

File: book/routing.rst (hunk 1 above)
Comment (javiereguiluz) [inline book/routing.rst]: Done. Thanks.

**Comment by WouterJ** [comment]
Please note tha twe have a cookbook article on this: http://symfony.com/doc/current/cookbook/routing/service_container_parameters.html I think we can reduce this tip directive to a small explanation and a link to the cookbook article. So for now, I'm a bit :-1:

**Comment by javiereguiluz** [comment]
I'm closing this because as @WouterJ showed, this would be a duplicated documentation.

**Comment by xabbuh** [comment]
Oh indeed, I completely missed that article.

@javiereguiluz Nonetheless, adding a link to the cookbook as suggest by Wouter sounds like a good idea to me.

**Comment by javiereguiluz** [comment]
@xabbuh done in #6506.

**Pull request closed without merging**